src/scripts/main.js

```diff
@@ -1 +1,63 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject('First promise was rejected');
+  }, 3000);
+});
```

Comment:
The first promise will reject after 3 seconds if no left mouse click occurs. This is correct, but ensure that this behavior is intended as it might lead to unexpected rejections if the user doesn't click in time.